### DISCUSSION TITLE
`Autoplay Video` 这种框删除了之后怎么恢复回来啊?

### DISCUSSION THREAD

**Discussion by sukanka** [AlistGo/alist#1641] [Q&A] [answered]
如图所示,实际上我已经删了好几个了,最开始是误删,然后居然找不到地方恢复.就又试了几下, 现在我的填 README 的框已经没有了,也找不到地方恢复.
 
![image](https://user-images.githubusercontent.com/31362108/189519744-daf8843c-6f68-41e1-9ee4-f4ddcfeda9c0.png)

![image](https://user-images.githubusercontent.com/31362108/189519784-0c7dbc7a-cb9c-4a66-a04d-42b6cca8de13.png)
---

版本 : 3.0.0-beta.1
安装方式: ` yay -S `[`alist-bin`](https://aur.archlinux.org/packages/alist-bin)

**Comment by chen8945** [comment, marked as answer]
这些带叉的多半是在 v3 已经被弃置的配置项，多数都能在其他选项中找到用于替代的配置项，就比如你图中的`Hide files`现在在`全局设置`中叫`隐藏文件`，其余选项同理
